org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/nexus/MandelbrotAcquireTest.java

```diff
@@ -81,7 +80,7 @@ private void checkNexusFile(IRunnableDevice<ScanModel> scanner) throws Exception
 		NXentry entry = rootNode.getEntry();
 		NXinstrument instrument = entry.getInstrument();
 
-		assertSolsticeScanGroup(entry, false, false);
+//		assertSolsticeScanGroup(entry, false, false, 1);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)